CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+# CHANGELOG
+
+## 0.5.0
+
+* Improvement - [sheltie: use /proc to find the bash binary](https://github.com/bottlerocket-os/bottlerocket-admin-container/pull/8) (#8, @samuelkarp)
```

Comment:
```suggestion
* Use /proc to find the bash binary in sheltie. ([#8])

[#8]: https://github.com/bottlerocket-os/bottlerocket-admin-container/pull/8
```

I recommend keeping the same voice and format as bottlerocket-os.

Reply:
Do you not need the PR link?  I thought we did in standalone markdown documents like changelogs.

Reply:
Sorry I missed that!